src/templates/navbar-template.js

```diff
@@ -83,8 +93,13 @@ export default function navbarTemplate() {
         </div>
       `
     }
+    <div style="display:flex; margin:12px 0; gap:8px; justify-content: start;">
+      <button class="m-btn thin-border" style="width:170px; margin: 0 auto;" part="btn btn-outline" @click='${(e) => {
+        downloadResource(this.specUrl, '35up-specs', e);
+      }}'>Download YAML file</button>
```

Comment:
```suggestion
      <a 
        download="35up-specs.yml"
        href=${this.specUrl}
        class="m-btn thin-border" style="width:170px; margin: 0 auto;" part="btn btn-outline"
      >
        Download YAML file
      </a>
```
Wouldn't this be better? That way you don't need a click handler. Is it to avoid the underlining of the text of the links?

Reply:
Fabian and Victor prefer to be a button instead of a link

Reply:
in a markup sense?

Reply:
I dont think they care about markup, link could be styled as a button